server/api.js

```diff
@@ -41,7 +42,22 @@ router.post("/correction", async (req, res) => {
 		});
 
 		if (completion.data.choices && completion.data.choices.length > 0) {
-			res.json(completion.data.choices[0].message.content);
+			const output = completion.data.choices[0].message.content;
+
+			// Store the output in the database
+			//await db.query("INSERT INTO history (output) VALUES ($1)", [output]);
+			// Store the output in the database
+			// Store the output in the database
+			try {
+				await db.query("INSERT INTO history (output) VALUES ($1)", [output]);
```

Comment:
what does this do? it only saves the output but not the user?

Reply:
I forgot to delete it. It was just for testing purposes before we decided to save the output after clicking the save button.